### PR TITLE
Update Cypress to ^4.8.0 and unpin on CI

### PR DESCRIPTION
**Pull request by simonsinclair**
Closes https://github.com/bbc/simorgh/pull/6269

### Overall change

In https://github.com/bbc/simorgh/pull/6269, using the latest (then 4.6.0) version of Cypress caused Jenkins to exit with a SIGTRAP error.

In Simorgh, the current version of Cypress (4.6.0) is now out of sync with CI (3.4.1). This difference is making debugging failing tests increasingly more difficult, as we are now seeing test fail on CI that otherwise don't fail locally.

### Code changes

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

### Testing

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by thekp** [comment]
Still breaks on CI with this log:
```
The Test Runner unexpectedly exited via a exit event with signal SIGTRAP
```

Same as this PR:https://github.com/bbc/simorgh/pull/6269 😭

**Comment by thekp** [comment]
We initially tried to bump cypress because we thought we required it for fixing the failing tests on this PR: https://github.com/bbc/simorgh/pull/6423

However, we have been able to fix the tests on that PR without bumping cypress so we are going to close to PR now since we hit a wall with this attempt to bump cypress.

**Summary of what was tried on this PR**: 
Cypress headless browser is electron, we suspected electron did not work as expected in our jenkins environment so we changed Cypress to run on Chromium which worked in Jenkins but Chromium is not available in our Travis CI so we tried running Cypress on Chrome in Travis however this did not work and exited unexpectedly with no clear error. 

TLDR: chromium headless browser works in jenkins, but not travis.
